Assets/Scripts/Model/GameManager.cs

```diff
@@ -1,17 +1,16 @@
 ﻿using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
-using UnityEngine.EventSystems;
 
 public class GameManager : MonoBehaviour
 {
     // SINGLETON
     public static GameManager Instance;
 
+    //GameObjects and variables
     public PlayerModel playerSouth;
     public PlayerModel playerNorth;
-    public PlayerModel currentPlayer; //player that has active turn
```

Comment:
this comments could staj, they are informative - espesially otherPlayer name might be misleading